stories/flow.stories.js

```diff
@@ -1,60 +1,35 @@
-import React from 'react'
+import React from "react";
 
-import { Flow } from '../src'
-import sample1 from './sample-data/sample.json'
-import sample2 from './sample-data/sample2.json'
-import sample3 from './sample-data/sample3.json'
-import sample4 from './sample-data/sample4.json'
+import { Flow } from "../src";
+import sample1 from "./sample-data/sample.json";
+import sample2 from "./sample-data/sample2.json";
+import sample3 from "./sample-data/sample3.json";
+import sample4 from "./sample-data/sample4.json";
```

Comment:
these are quite unnecessary. i'll ensure the linter to also govern the stories folder in a follow up PR, but can we undo these non-feature changes especially they're not following our conventions (like the one used in react-labs)?

if time's a factor, don't worry about this and i'll handle it in a follow up PR